src/ucp/wireup/select.c

```diff
@@ -103,6 +103,7 @@ typedef struct {
 static const char *ucp_wireup_md_flags[] = {
     [ucs_ilog2(UCT_MD_FLAG_ALLOC)]               = "memory allocation",
     [ucs_ilog2(UCT_MD_FLAG_REG)]                 = "memory registration",
+    [ucs_ilog2(UCT_MD_FLAG_RKEY_PTR)]            = "obtain remote memory pointer"
```

Comment:
shouldn't other flags be initialized as well?

Reply:
If I'm not mistaken, other flags don't participate in selection logic.
@yosefe @brminich do we need to add:
- `UCT_MD_FLAG_ADVISE`
- `UCT_MD_FLAG_FIXED`
- `UCT_MD_FLAG_SOCKADDR`?

Other flags just inform the local user if there are any restrictions (e.g. `UCT_MD_FLAG_NEED_MEMH` requires providing a memory handle for a local buffer)